app/code/Magento/AdvancedPricingImportExport/composer.json

```diff
@@ -15,6 +15,9 @@
         "magento/module-import-export": "100.3.*",
         "magento/module-store": "100.3.*"
     },
+    "require-dev": {
```

Comment:
Could you explain why `require-dev` section is needed?

Reply:
In general - for development purposes only, not used in production.
In our case, we agreed to use this section as an indicator that PHP 7.2 support is experimental yet, and should not be used in production. But looks like Composer doesn't really like an idea of having the same dependency being declared in two places, so we might need to reconsider the solution.